shell/platform/glfw/public/flutter_glfw.h

```diff
@@ -167,6 +175,13 @@ FLUTTER_EXPORT void FlutterDesktopWindowSetPixelRatioOverride(
     FlutterDesktopWindowRef flutter_window,
     double pixel_ratio);
 
+// Sets the min/max size of |flutter_window| in pixels. Use
```

Comment:
s/pixels/screen coordinates/ for consistency with the other window size APIs (pixels is not necessarily true).